pages/donate.js

```diff
@@ -0,0 +1,52 @@
+// Component imports
+import Page from '../components/Page'
+
+
+
+
+
+// Component constants
+const title = 'Donate'
+
+
+
+
+
+const Donate = () => (
+  <div className="page-wrapper">
+    <header className="page-header">
+      <h1>{title}</h1>
+    </header>
+
+    <div className="page-content">
+      <p>
+        The Fuel Rats are an Elite group of pilots who specialize in saving stranded CMDRs from the pitch black abyss of space.
+        <br />
+        Since 3301, We've built up a plethora of custom tools and services to assist in meeting our ever-growing demand.
+        <br />
+        With growth, has come expenses. The cost to keep our lights on has grown significantly, and now exceed €100 per month.
```

Comment:
"With growth has come expenses." flows a bit better; not sure a comma is needed there.